lxd/device/disk.go

```diff
@@ -691,6 +754,13 @@ func (d *disk) postStop() error {
 		}
 
 	}
+	if strings.HasPrefix(d.config["source"], "ceph") {
```

Comment:
This chunk must be moved to run last in `postStop` otherwise, it'll be causing the test errors we've seen:
```
EROR[11-19|02:54:01] Failed to stop device 'rbd': Failed to run: rbd unmap /dev/rbd3: rbd: sysfs write failed
rbd: unmap failed: (16) Device or resource busy 
```